src/app/App.tsx

```diff
@@ -8,11 +8,29 @@ import { znTheme } from "./shared/style/Style-MainTheme";
 
 const theme = znTheme;
 
+function detectMobileDevice(): boolean {
+    const userAgent = navigator.userAgent.toLowerCase();
+    const isMobile = /iphone|ipod|android|ie|blackberry|fennec/.test(userAgent);
+    return isMobile;
+}
+
+function App(): ReactElement {
+    useEffect(() => {
+        if (detectMobileDevice()) {
+            alert(
+                "Please note: The Zoonotify Webapp has not been optimized for mobile screens."
```

Comment:
This should be a TOKEN with German and English translation provided by the language files

Reply:
ok